src/PRDevice.cpp

```diff
@@ -71,15 +71,15 @@ PRDevice* PRDevice::Create(PRMachineType machineType)
     if (machineType != kPRMachineCustom && machineType != kPRMachinePDB &&
```

Comment:
The majority of changes in this file are to remove whitespace and get rid of variables used to hold return values that are then ignored.  I called out one notable change in a comment below, and there are multiple additions of `PRSetLastErrorText()` to improve error reporting.